From 40737606bbbb382e7cd02a42da0526a3741c9b3e Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Fri, 5 Jan 2018 19:40:16 +0800
Subject: [PATCH 30/35] arm64: allwinner: h6: add CPU regulator for Pine H64

Pine H64 uses the DCDCA (poly-phased with DCDCB) output of AXP805 PMIC
to power the ARM cores in it.

Add the regulator to the CPU device node.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
index 038e213..7f8e958 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
@@ -47,6 +47,10 @@
 	};
 };
 
+&cpu0 {
+	cpu-supply = <&reg_dcdca>;
+};
+
 &dwc3 {
 	status = "okay";
 };
-- 
2.7.4

